app/javascript/dashboard/components/ui/TimeAgo.vue

```diff
@@ -27,17 +27,31 @@ export default {
   },
   data() {
     return {
-      lastActivityAtTimeAgo: dynamicTime(this.lastActivityTimestamp),
-      createdAtTimeAgo: dynamicTime(this.createdAtTimestamp),
+      lastActivityAtTimeAgo: dynamicTime(
+        this.lastActivityTimestamp,
+        this.$root.$i18n.locale
+      ),
+      createdAtTimeAgo: dynamicTime(
+        this.createdAtTimestamp,
+        this.$root.$i18n.locale
+      ),
```

Comment:
I don't see the changes for `dynamicTime` function, same with `shortTimestamp` did you forget the commit it?